### PR TITLE
Build static storybooks utility script

### PR DESCRIPTION
**Pull request by shilman**
Issue: N/A

## What I did

To run chromatic on local machine, you should build all the static storybooks first

## How to test

```
yarn build-storybooks
yarn chromatic
```

Is this testable with Jest or Chromatic screenshots?
Does this need a new example in the kitchen sink apps?
Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

### CODE REVIEW

**Comment by tmeasday** [comment]
LGTM. Should we use it in the CI scripts?

**Review by tmeasday — APPROVED** [review]

**Comment by Hypnosphi** [comment]
@tmeasday we build different frameworks in parallel now and I think it's good as is

**Comment by Hypnosphi** [comment]
> Please check if this works on windows

I won't mind even if it doesn't. This helper looks like a pretty optional thing

File: scripts/build-storybooks.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# This builds all the example storybooks for running chromatic on your dev machine
+
+examples=(angular-cli cra-kitchen-sink html-kitchen-sink mithril-kitchen-sink polymer-cli vue-kitchen-sink official-storybook)
```
Comment (Hypnosphi) [inline scripts/build-storybooks.sh]: marko?

**Pull request merged**